apps/api-extractor/src/schemas/api-extractor-template.json

```diff
@@ -175,7 +176,16 @@
      * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
      * DEFAULT VALUE: "<projectFolder>/temp/"
      */
-    // "reportTempFolder": "<projectFolder>/temp/"
+    // "reportTempFolder": "<projectFolder>/temp/",
+
+    /**
+     * Whether "forgotten exports" should be included in the API report file. Forgotten exports are declarations
+     * flagged with `ae-forgotten-export` warnings. See https://api-extractor.com/pages/messages/ae-forgotten-export/ to
```

Comment:
We have historically avoided using `` ` `` or other markdown inside JSON config file comments.  Editors don't highlight it, and it's unclear whether TSDoc can be used there.  But we do use `/**` comment framing, which I suppose is confusing.

🤔 I suppose we should either (1) decide to adopt TSDoc in JSON config files, or else (2) replace `/**` with `/*` to clarify that they are plain text comments.  Let me know your opinion. I'll ask some other maintainers as well.

Reply:
Consensus is that we should allow TSDoc in config files. Don't worry about it too much in this PR -- we'll make a separate PR to tune up the `/**` syntax in config files